src/modules/register/register-form/steps/confirm-account/ConfirmAccountImage.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { memo } from 'react';
+import React from 'react';
 
-export const ConfirmAccountImage = memo(() => {
+export const ConfirmAccountImage = () => {
```

Comment:
Why you're not memo'izing icons?

Reply:
Because i make performance tests with browser profiler and memo usage gives worst result - because always when we trying to use memo we should perform tests - before and after.